homeassistant/components/trello/sensor.py

```diff
@@ -0,0 +1,82 @@
+"""Platform for sensor integration."""
+from __future__ import annotations
+
+from homeassistant.components.sensor import SensorEntity, SensorStateClass
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.device_registry import DeviceEntryType, DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, Board, List
+from .coordinator import TrelloDataUpdateCoordinator
+
+
+class TrelloSensor(CoordinatorEntity[TrelloDataUpdateCoordinator], SensorEntity):
```

Comment:
If I understood this sensor correctly, it will show the number of entries of the board.
If this is correct please use a [todo entity](https://developers.home-assistant.io/docs/core/entity/todo) instead

Reply:
So technically, no, it's the number of "cards" on a list.

You're definitely right, though, that Trello shares a lot of similarities with to-do functionality. However, exactly how Trello can be used as a to-do list can vary significantly by user and I don't want to assume user's workflows.

There's a [discussion about this that goes into more detail on the custom component repo](https://github.com/ScottG489/ha-trello/issues/3) if you're interested to read in more detail.

Essentially since Trello boards are Kanban boards, it's a lot more open ended how the user can use it vs a traditional to-do list, but let me know what you think!

Reply:
I'm not familiar with Trello, but you could create a todo list per board and set the [supported_features](https://developers.home-assistant.io/docs/core/entity/todo#supported-features). We don't support moving a entry from one list to another but a todo list shows more info than a sensor. A user could even add new entry to one list
What do you think?

Reply:
I think it's worth considering, but there are a lot of different ways it could be implemented.

What do you think we should do when a card is created in the to-do list? The Trello board itself has multiple lists, so it seems arbitrary as to where the card created from the to-do list item would go.

Also what are your thoughts on how to mark a card as done in Trello? I'm not sure how this concept translates from a to-do list to a Trello board.

I'm also not sure about re-ordering cards since the cards would be split across multiple Trello lists. How do you think we should translate that to a single to-do list in HA? That's just something we could mark as not supported via `supported_features` as you said, though.

Reply:
> The Trello board itself has multiple lists,

Can we have a todo list per board? Not sure if this is usable and sorry I'm not using Trello and so it's hard to suggestion the correct way

Reply:
Apologies for my delay getting back to this!

So yeah I think it just comes down to the fact that a board doesn't translate well enough into a to-do list unfortunately.

Reply:
I discussed this in a call with @edenhaus and a few others, the conclusion is that it's fine to model this as a sensor 👍